### PR TITLE
Remove electron from WATcher

### LINKED ISSUE #102: Removal of Electron
As mentioned in https://github.com/CATcher-org/WATcher/pull/84#issuecomment-1475654240, let's remove electron from WATcher as well.

### PR DESCRIPTION
**Pull request by vigneshsankariyer1234567890**
### Summary:

Resolves #102. 

Currently, WATcher does not use Electron, with there being no need to use a desktop application as a back-up. Furthermore, Electron has been completely removed from CATcher, and being a fork of CATcher, WATcher should have its dependencies updated as well.

This PR follows the steps that were taken in the corresponding PR for [CATcher](https://github.com/CATcher-org/CATcher/pull/1153) to remove instances of Electron from its codebase.

### Changes Made:
- Removed ElectronService from ./src
- Removed all uses of electron library in ./src
- Removed scripts that use electron such as main.ts and /electron-utils
- Removed scripts from package.json that use electron
- Removed packages from package.json such as electron, electron-builder, electron-logger, etc.

### Commit Message:
```
Refactor and remove uses of Electron from WATcher

There are many uses of Electron in WATcher, even with the CATcher Electron app being deprecated. 

As such, there are many instances of unused code, which is bad practice.

Removing instances of uses of Electron allows us to trim the codebase and remove dead code.

Let's 
* Refactor and remove uses of Electron from WATcher
* Remove all uses of Electron in the `src` folder.
* Remove uses of Electron from `package.json` and other files
* Delete folders such as `electron-utils` and files such as `postinstall` that are used to initialise electron
* Remove and amend scripts in `package.json` that are involved in building the electron app
* Edit `deployment-actions.yml` to remove pre-build actions from CD workflow
```

### CODE REVIEW

File: tests/services/logging.service.spec.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ const mockDates = () => {
 const initializeLoggingService = () => {
   const electronService = jasmine.createSpyObj('ElectronService', ['isElectron']);
```
Comment (kkangs0226) [inline tests/services/logging.service.spec.ts:23]: Can we remove this?

**Review by kkangs0226 — CHANGES_REQUESTED** [review]
image
There is an error when I try to open an issue, please look into it!

File: tests/services/logging.service.spec.ts (hunk 1 above)
Comment (vigneshsankariyer1234567890) [reply tests/services/logging.service.spec.ts:23]: yes, seems like i forgot to remove it, i'll remove it now

**Comment by vigneshsankariyer1234567890** [comment]
> There is an error when I try to open an issue, please look into it!

@kkangs0226 found the error, seems like I forgot to add a line to open the issues page in the browser! Thank you for catching it.

**Comment by vigneshsankariyer1234567890** [comment]
@kkangs0226 @chunweii just bumping again for a re-review!

**Review by kkangs0226 — APPROVED** [review]
LGTM!

**Review by chunweii — CHANGES_REQUESTED** [review]
The electron service in `confirm-login.component.spec.ts` is not removed. Please remove it.

**Review by chunweii — APPROVED** [review]

**Pull request merged**